### ISSUE TITLE
[Experiment] TODO

### ISSUE THREAD

**Issue by ahnjaeshin** [yuyupopo/DeepPano#5]
# Hyper-parameters
1. Model Type : UNet, 티라미수 ~~먹고싶다~~  
2. Model Layer 수 : 20 layer => 30 layer  => model capacity에 기여  
3. Upsample : bilinear vs transposed convolution => model capacity에 기여  
4. Loss function : Binary Cross Entropy vs Differentiable IOU vs differentiable DICE
5. Initialization type : none vs he init
6. dropout: none vs all layer vs only last layer vs only at middle

------------------------ mid term

7. augmentation : rotation

# Data
1. Only Segmentable vs All
2. only front teeth
3. Data size : 500 vs 750
4. margin type 2 ~7

# Metric
1. DICE
2. Precision & Recall

# Experiments
1. Init 유무
2. **train / val input size 다름**
3. 일반 unet -> 우리꺼

-------------------------------------------------

> From experiments of all above, 

> layer number : convolution layer 기준
